cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tether/TetherUtils.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.internal.tether;
+
+import com.google.common.net.HttpHeaders;
+import io.cdap.cdap.common.internal.remote.RemoteAuthenticator;
+import io.cdap.common.http.HttpMethod;
+import io.cdap.common.http.HttpRequestConfig;
+import io.cdap.common.http.HttpRequests;
+import io.cdap.common.http.HttpResponse;
+
+import java.io.IOException;
+import java.net.URI;
+import javax.annotation.Nullable;
+
+/**
+ * Utility functions for tethering.
+ */
+public class TetherUtils {
+  // Connection timeout = 5 seconds.
+  private static final int TIMEOUT_MS = 5000;
+  private static RemoteAuthenticator authenticator = RemoteAuthenticator.getDefaultAuthenticator();
```

Comment:
static final. Also need to make it all caps

Reply:
fixed